### DISCUSSION TITLE
How to get context?

### DISCUSSION THREAD

**Discussion by Roc-00** [open-telemetry/opentelemetry-java#3618] [Q&A]
I'm a novice using OLE. I'm trying to set trace in the handler of netty. I read the example of HTTP(https://github.com/open-telemetry/opentelemetry-java/tree/main/examples/http)
But I'm not sure if I'm doing it right
this is my code:
![image](https://user-images.githubusercontent.com/84842773/133255413-6cc4ea3e-c5ef-45d4-98c6-c4f90d3191b1.png)

**Comment by jkwatson** [comment]
I'm not sure what "OLE" is, but this does look like it should work. Is it not?

**Reply by Roc-00** [reply]
I misspelled it,it was 'otle'.I haven't finished the code yet,I'm not sure how to replace ‘keySet ()’ to achieve the same purpose as in example.But I'm going in the right direction, right？

**Reply by jkwatson** [reply]
This looks like the right direction, yes. The experts in netty instrumentation work mostly in the opentelemetry-java-instrumentation repository, so it's probably worth asking over there for detailed help beyond the basics.

**Reply by Roc-00** [reply]
OK,thank you
